core/base/integralLines/IntegralLines.h

```diff
@@ -190,17 +194,20 @@ int ttk::IntegralLines::execute() const {
 
   {
     std::stringstream msg;
-    msg << "[IntegralLines] Data-set (" << vertexNumber_
-        << " points) processed in " << t.getElapsedTime() << " s. ("
-        << threadNumber_ << " thread(s))." << std::endl;
-    dMsg(std::cout, msg.str(), timeMsg);
+    msg << "Data-set (" << vertexNumber_ << " points) processed in "
+        << t.getElapsedTime() << " s. (" << threadNumber_ << " thread(s)).";
+    this->printMsg(msg.str());
```

Comment:
see https://github.com/topology-tool-kit/ttk/blob/dev/core/base/helloWorld/HelloWorld.h:123 for an example of time logging